skimage/restoration/_denoise.py

```diff
@@ -10,39 +9,89 @@
 import numbers
 
 
-def _gaussian_weight(sigma_sqr, value, *, dtype=float):
-    return np.exp(-0.5 * value * value / sigma_sqr, dtype=dtype)
+def _gaussian_filter(array, sigma_squared, *, dtype=float):
+    """Helping function. Define a Gaussian filter from array and
+    sigma_square.
+
+    Parameters:
+    -----------
+    array : ndarray
+        Input array.
+    sigma_squared : float
+        The squared standard deviation used in the filter.
+    dtype : data type object
+        The type and size of the data to be returned.
+
+    Returns:
+    --------
+    gaussian : ndarray
+        The input array filtered by the Gaussian.
+    """
+    return np.exp(-0.5 * (array ** 2  / sigma_squared), dtype=dtype)
 
 
 def _compute_color_lut(bins, sigma, max_value, *, dtype=float):
-    values = np.linspace(0, max_value / bins, bins, endpoint=False)
-    color_lut = _gaussian_weight(sigma**2, values, dtype=dtype)
-    return color_lut
+    """Helping function. Define a lookup table containing Gaussian filter
+    values using the color distance sigma.
+
+    Parameters:
+    -----------
+     bins : int
+        Number of discrete values for Gaussian weights of color filtering.
+        A larger value results in improved accuracy.
+    sigma : float
+        Standard deviation for grayvalue/color distance (radiometric
+        similarity). A larger value results in averaging of pixels with larger
+        radiometric differences. Note, that the image will be converted using
+        the `img_as_float` function and thus the standard deviation is in
+        respect to the range ``[0, 1]``. If the value is ``None`` the standard
+        deviation of the ``image`` will be used.
+    max_value : float
+        Maximum value of the input image.
+    dtype : data type object
+        The type and size of the data to be returned.
 
+    Returns:
+    --------
+    color_lut : ndarray
+        Lookup table for the color distance sigma.
+    """
+    values = np.linspace(0, max_value, bins, endpoint=False)
+    return _gaussian_filter(values, sigma**2, dtype=dtype)
```

Comment:
I don't think you should rename `_gaussian_weights`. This is not a filter, it's a weighting.